testbeam_analysis/dut_alignment.py

```diff
@@ -779,16 +757,16 @@ def apply_alignment_to_chunk(hits_chunk, dut_index, use_prealignment, alignment,
                         if use_duts is not None and dut_index not in use_duts:  # omit DUT
                             continue
 
-                        apply_alignment_to_chunk(hits_chunk=hits_chunk, dut_index=dut_index, use_prealignment=use_prealignment, alignment=prealignment if use_prealignment else alignment, inverse=inverse, no_z=no_z)
+                        geometry_utils.apply_alignment_to_hits(hits=hits_chunk, dut_index=dut_index, use_prealignment=use_prealignment, alignment=alignment, inverse=inverse, no_z=no_z)
 
                     hits_aligned_table.append(hits_chunk)
                     progress_bar.update(index)
                 progress_bar.finish()
 
     logging.debug('File with realigned hits %s', output_hit_file)
 
-
-def alignment(input_track_candidates_file, input_alignment_file, n_pixels, pixel_size, align_duts=None, selection_fit_duts=None, selection_hit_duts=None, selection_track_quality=1, initial_rotation=None, initial_translation=None, max_iterations=10, use_n_tracks=200000, plot=False, chunk_size=100000):
+# TODO: selection_track_quality to selection_track_quality_sigma
+def alignment(input_merged_file, input_alignment_file, n_pixels, pixel_size, dut_names=None, use_prealignment=True, align_duts=None, align_telescope=None, selection_fit_duts=None, selection_hit_duts=None, quality_sigma=5.0, alignment_order=None, initial_rotation=None, initial_translation=None, max_iterations=3, max_events=100000, use_fit_limits=True, new_alignment=True, plot=False, chunk_size=100000):
```

Comment:
I do not like to distinguish `duts` from `telescope` because what is a `dut` is is defined in result analysis and what a `telescope` (plane) is is defined in fit tracks. So why to start to distinguish here?